pkg/vm/emit/emit.go

```diff
@@ -156,13 +157,19 @@ func AppCall(w *io.BinWriter, scriptHash util.Uint160) {
 	Syscall(w, interopnames.SystemContractCall)
 }
 
-// AppCallWithOperationAndArgs emits an APPCALL with the given operation and arguments.
-func AppCallWithOperationAndArgs(w *io.BinWriter, scriptHash util.Uint160, operation string, args ...interface{}) {
+// AppCallWithFlag emits an APPCALL with the full set of parameters.
+func AppCallWithFlag(w *io.BinWriter, scriptHash util.Uint160, operation string, f flags.CallFlag, args ...interface{}) {
 	Array(w, args...)
+	Int(w, int64(f))
 	String(w, operation)
 	AppCall(w, scriptHash)
 }
 
+// AppCallWithOperationAndArgs emits an APPCALL with the default parameters given operation and arguments.
+func AppCallWithOperationAndArgs(w *io.BinWriter, scriptHash util.Uint160, operation string, args ...interface{}) {
```

Comment:
`AppCallAllFlag`? Maybe it's easier to have one `AppCall`?

Reply:
Replaced this with `AppCall` and `AppCallNoArgs`. Latter is used in multiple tests and `rpc`, when arguments are already on stack.